tests/ModelParser/PhpDocParserTest.php

```diff
@@ -204,7 +229,7 @@ public function testInheritedProperty(): void
         $this->assertPropertyCollection('parent_property2', 1, $props[1]);
         $property = $props[1]->getVariations()[0];
         $this->assertProperty('parentProperty2', false, false, $property);
-        $this->assertPropertyType(PropertyTypePrimitive::class, 'bool', false, $property->getType());
+        $this->assertPropertyType(PropertyTypePrimitive::class, 'string', false, $property->getType());
```

Comment:
this has been a bug: when overwriting the same property with a different annotation, we want the overwriting annotation to win, not the one of the base class.